### ISSUE TITLE
unable to use actix-web@2.0 with mongodb river@0.9.1

### ISSUE THREAD

**Issue by 0x8f701** [actix/actix-web#1407]
Steps to reproduce:
```
cargo new test_actix_mongo
cd test_actix_mongo
cargo add actix-web
cargo add mongodb
cargo run
```

errors:
```
    Updating crates.io index
error: failed to select a version for `trust-dns-proto`.
    ... required by package `actix-connect v1.0.1`
    ... which is depended on by `actix-http v1.0.1`
    ... which is depended on by `actix-web v2.0.0`
    ... which is depended on by `test_mongo_actix-web v0.1.0 (/home/cheng/Projects/Rust/test_mongo_actix-web)`
versions that meet the requirements `= 0.18.0-alpha.2` are: 0.18.0-alpha.2

all possible versions conflict with previously selected packages.

  previously selected package `trust-dns-proto v0.18.0`
    ... which is depended on by `mongodb v0.9.1`
    ... which is depended on by `test_mongo_actix-web v0.1.0 (/home/cheng/Projects/Rust/test_mongo_actix-web)`

failed to select a version for `trust-dns-proto` which could resolve this conflict
Error: cargo metadata returned exit code: 101
```

**Comment by JohnTitor** [comment]
It isn't actix-web issue but actix-connect one and already fixed in 2.0.0-alpha.2.

**Comment by 0x8f701** [comment]
@JohnTitor  Hello, shall a new version of actix-web be release for fixing this?

**Comment by JohnTitor** [comment]
@GopherJ Yeah, we're releasing some alpha versions now but actix-web is yet. Okay, I'll make a release for actix-web in a few days.

**Comment by JohnTitor** [comment]
@GopherJ Just released actix-web v3.0.0-alpha.1. Let me know if you also want to release other extra crates.
